CONTRIBUTING.md

```diff
@@ -1,9 +1,9 @@
 # Contributing to Kong :monkey_face:
 
 Hello, and welcome! Whether you are looking for help, trying to report a bug,
-thinking about getting involved in the project, or about to submit a patch, this
+thinking about getting involved in our projects, or about to submit a patch, this
```

Comment:
Please change this back, this document only concerns itself with a single project - the Kong Gateway.

```suggestion
thinking about getting involved in the project, or about to submit a patch, this
```